doc/antora/modules/howto/nav.adoc

```diff
@@ -10,7 +10,7 @@
 **** xref:modules/ldap_search.adoc[Testing with ldapsearch]
 **** xref:modules/ldap_configuration.adoc[Configuration]
 **** xref:modules/ldap_docker.adoc[Docker]
-**** xref:modules/ldap_authentication.adoc[Authenticating Users]
+**** xref:ROOT:modules/ldap_authentication.adoc[Authenticating Users]
```

Comment:
It's best to just delete this, and then have the  ldap "howto" pages point to the concepts page.

The navbar here should be only for things within this particular module.